File: my-code/0009-palindrome-number/solution.c
bool isPalindrome(int x) {
    int count=0; 
    int n=x;
    if(x<0)
    {
        return false;
    }
    if(x<10)
    {
        return true;
    }
    int p=0;
    while(n>0)
    {
       n=n/10;
       p++;
    }
    int b[p];
    if(x>=10)
    {
        while(x>0)
        {
            b[count]=x%10;
            x=x/10;
            count++;
        }
    }
    int c[count];
    for(int i=0;i<count;i++)
    {
        c[i]=b[count-i-1];
    }
    for(int i=0;i<count;i++)
    {
        if(b[i]!=c[i])
        {
            return false;
        }
        if(i==count-1)
        {
            return true;
        }
    }
    return 0;
}

File: my-code/0009-palindrome-number/solution.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if(x<0):
            return False
        else:
            y= len(str(x))
            z=str(x)
            for i in range(y//2):
                if(z[i]!=z[y-i-1]):
                    return False
            return True



